arelle/RuntimeOptions.py

```diff
@@ -0,0 +1,163 @@
+    formulaUnsatisfiedAsser: Optional[bool] = None
+    formulaUnsatisfiedAsserError: Optional[bool] = None
+    formulaVarExpressionCode: Optional[bool] = None
+    formulaVarExpressionEvaluation: Optional[bool] = None
+    formulaVarExpressionResult: Optional[bool] = None
+    formulaVarExpressionSource: Optional[bool] = None
+    formulaVarFilterWinnowing: Optional[bool] = None
+    formulaVarFiltersResult: Optional[bool] = None
+    formulaVarSetExprEval: Optional[bool] = None
+    formulaVarSetExprResult: Optional[bool] = None
+    formulaVarsOrder: Optional[bool] = None
+    formulaeFile: Optional[str] = None
+    httpUserAgent: Optional[str] = None
+    httpsRedirectCache: Optional[bool] = None
+    importFiles: Optional[str] = None
+    infosetValidate: Optional[bool] = None
+    internetConnectivity: Optional[str] = None
+    internetLogDownloads: Optional[bool] = None
+    internetRecheck: Optional[str] = None
+    internetTimeout: Optional[int] = None
+    keepOpen: Optional[bool] = None
+    labelLang: Optional[str] = None
+    labelRole: Optional[str] = None
+    logCodeFilter: Optional[str] = None
+    logFile: Optional[str] = None
+    logFormat: Optional[str] = None
+    logLevel: Optional[str] = None
+    logLevelFilter: Optional[str] = None
+    logRefObjectProperties: Optional[str] = None
+    logTextMaxLength: Optional[int] = None
+    monitorParentProcess: Optional[bool] = None
+    noCertificateCheck: Optional[bool] = None
+    outputAttribution: Optional[str] = None
+    packageManifestName: Optional[str] = None
+    packages: Optional[str] = None
+    parameterSeparator: Optional[str] = None
+    parameters: Optional[str] = None
+    password: Optional[str] = None
+    plugins: Optional[str] = None
+    preFile: Optional[str] = None
+    proxy: Optional[str] = None
+    relationshipCols: Optional[int] = None
+    roleTypesFile: Optional[str] = None
+    rssReport: Optional[str] = None
+    rssReportCols: Optional[int] = None
+    showEnvironment: Optional[bool] = None
+    showOptions: Optional[bool] = None
+    skipDTS: Optional[bool] = None
+    skipLoading: Optional[bool] = None
+    statusPipe: Optional[str] = None
+    tableFile: Optional[str] = None
+    testReport: Optional[str] = None
+    testReportCols: Optional[int] = None
+    testcaseResultOptions: Optional[str] = None
+    testcaseResultsCaptureWarnings: Optional[bool] = None
+    timeVariableSetEvaluation: Optional[bool] = None
+    uiLang: Optional[str] = None
+    username: Optional[str] = None
+    utrUrl: Optional[str] = None
+    utrValidate: Optional[bool] = None
+    validate: Optional[bool] = None
+    validateEFM: Optional[bool] = None
+    validateEFMCalcTree: Optional[bool] = None
+    validateHMRC: Optional[bool] = None
+    validateTestcaseSchema: Optional[bool] = None
+    versReportFile: Optional[str] = None
+    viewArcrole: Optional[bool] = None
+    viewer_feature_review: Optional[bool] = False
```

Comment:
This is the (not base Arelle) ixbrl-viewer option, isn't it? Also, I think you're missing [logFileMode](https://github.com/Arelle/Arelle/blob/5d4e2515a9edb33e1da61ca9013c62353e62918a/arelle/CntlrCmdLine.py#L216)